charts/alertmanager/values.yaml

```diff
@@ -168,9 +168,18 @@ persistence:
     - ReadWriteOnce
   size: 50Mi
 
+configAnnotations: {}
+  ## For example if you want to provide private data from a secret vault
+  ## https://github.com/banzaicloud/bank-vaults/tree/main/charts/vault-secrets-webhook
+  ## P.s.: Add option `configMapMutation: true` for vault-secrets-webhook
+  # vault.security.banzaicloud.io/vault-role: "admin"
+  # vault.security.banzaicloud.io/vault-addr: "https://vault.vault.svc.cluster.local:8200"
+  # vault.security.banzaicloud.io/vault-skip-verify: "true"
+  # vault.security.banzaicloud.io/vault-path: "kubernetes"
+
 config:
   global: {}
-    # slack_api_url: ''
+    # slack_api_url: '${vault:secret/data/slack-hook-alerts#URL}'
```

Comment:
This seems unintended

Reply:
No, it's intentional, use case.

Reply:
most people will not use this and therefore i woulld not add this as default example.